ssc/cmod_mhk_tidal.cpp

```diff
@@ -169,32 +169,49 @@ class cm_mhk_tidal : public compute_module
         min_velocity = tidal_resource_matrix.at(0, 0);
         max_velocity = tidal_resource_matrix.at(size_t(number_rows) - 1, 0);
 
-		for (int i = 0; i < number_rows; i++) {
+        for (int i = 0; i < number_rows; i++) {
             size_t n = i;
-            if (tidal_resource_matrix.at(n, 1) != 0 && tidal_resource_matrix.at(n - 1, 1) == 0 && n != 0)
-            {
-                tidal_resource_start_velocity = tidal_resource_matrix.at(n, 0);
+            if (i != 0) {
+                if (tidal_resource_matrix.at(n, 1) != 0 && tidal_resource_matrix.at(n - 1, 1) == 0)
```

Comment:
Can you explain these changes in the pull request title? I'm all for fixing tests, but this seems a little out of place in this PR.